nix/haskell-pins.nix

```diff
@@ -182,12 +196,12 @@ let
         sha256 = "sha256-8FM3IAA3ewCuv9Mar8aWmzbyfKK9eLXIJPMHzmYb1zE=";
       };
     };
-    # This can be removed once postie 0.6.0.3 (or later) is in nixpkgs
+    # This can be removed once postie with TLS 1.9 is on nixpkgs.
     postie = {
       src = fetchgit {
-        url = "https://github.com/alexbiehl/postie.git";
-        rev = "c92702386f760fcaa65cd052dc8114889c001e3f";
-        sha256 = "sha256-yiw6hg3guRWS6CVdrUY8wyIDxoqfGjIVMrEtP+Fys0Y=";
+        url = "https://github.com/wireapp/postie.git";
+        rev = "43b6d1d21d56e567077c194d49efb92e777e7628";
+        sha256 = "sha256-DKugy4EpRsSgaGvybdh2tLa7HCtoxId+7RAAAw43llA=";
```

Comment:
Is there an open PR for this? (this is our own fork no?)

Reply:
https://github.com/alexbiehl/postie/pull/4